index.js

```diff
@@ -46,6 +47,10 @@ module.exports = (source, destination, options = {}) => {
 		let files;
 		try {
 			files = await globby(source, options);
+
+			if (options.ignoreJunk) {
+				files = files.filter(junk.not);
```

Comment:
`junk.not` accepts **filenames**, but `globby` produces **file paths**, which can be filenames or longer deeper paths. I think you need some more tests.